### PR TITLE
Added new levels to the game

### LINKED ISSUE #6: add level to the game
currently, the game only has a 4 X 3 board, add a 4X5 and 4X7 as medium and hard levels respectively

### PR DESCRIPTION
**Pull request by harshN-17**
fixed #6 
added two new levels to the game. 
used 6 new images for the same.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -27,24 +51,44 @@ function restartGame(){
...
+        'images/cards/kimetsu.png',
+        'images/cards/kimetsu.png',
+        'images/cards/nezuko2.png',
+        'images/cards/nezuko2.png',
+        //need upto 24 cards for level 3
+        'images/cards/tokito.png',
+        'images/cards/tokito.png',
+        'images/cards/genya.png',
+        'images/cards/genya.png',
     ];
```
Comment (dev-lovedeep) [inline index.js:81]: in this Array make the path appear only one time, and then where you are generating cardArray, generate this type of pattern there

File: index.html [hunk 2]
```diff
@@ -22,8 +22,9 @@ <h4 class="time--stat" style="text-align:center">
         time:<span id="timer">0</span>
       </span>
     </h3>
-    <section class="memory-game">
-        <div class="memory-card">
+    <section 
+    class="memory-game">
+        <!-- <div class="memory-card">
           <img class="front-face" src="#" alt="card front face" />
```
Comment (dev-lovedeep) [inline index.html:28]: dont leave the code commented, just leave one of the them , so anyone else can understand the structure

File: index.js [hunk 3]
```diff
@@ -4,6 +4,30 @@ function restartGame(){
 
 // Wrapping entire code in anonymous function and calling it, so that user doesn't have access to cardImageSrcs
 (() => {
+    const game = document.querySelector('.memory-game');
+  function openM(){
+    alert("man");
+  }
```
Comment (dev-lovedeep) [inline index.js:10]: why is this used

Comment (harshN-17) [reply index.js]: > why is this used

this was not coded by me but i have removed it

**Comment by harshN-17** [comment]
i have made a new commit by updating and resolving all the issues

**Review by dev-lovedeep — COMMENTED** [review]
Changes look good, only one thing in the main branch some changes were merged(like sounds were added) merge those changes with yours and then we can close this PR

**Comment by harshN-17** [comment]
i have combined the updates with my code. the final commit is ready to be merged

**Pull request merged**